### DISCUSSION TITLE
Need help with schedules

### DISCUSSION THREAD

**Discussion by Lachney** [pterodactyl/panel#2861] [Support] [answered]
**Panel version**: 1.1.3
**Wings version**: 1.1.3
**Distro**: Ubuntu 20.04
**Web Server**:: Nginx

I've done a lot of searching in the Pterodactyl discord and I found a couple of others solutions, but none of it seems to help in my case.

**The issue**
I can create a schedule and add tasks. However, it does not run any of the schedules automatically for me. Forcing the schedule to run by clicking "Run now" does work however. I've tried out a schedule that runs every minute but it just doesn't run at all.

**Things I've tried**
1. Made sure that the crontab is setup.
2. Made my servers timezone match with pterodactyl and wings with "America/Chicago".
3. Ran wings in debug and didn't see any errors or warnings.

I'm not sure if I'm doing it correctly, but I cannot seem to find anyone else with this issue or even know if anyone has schedules working. I'd appreciate it if someone could please guide me in the right direction. Thanks.

**Comment by DaneEveritt** [comment, marked as answer]
Your queue worker (pteroq) isn't running.

**Reply by Lachney** [reply]
Thanks Dane, I didn't think to check there. I found the problem. I have Pterodactyl setup in a different directory so I had to modify the directory it was looking for in `ExecStart`.

However I'm still getting the same issue, none of the schedules are going through. Anything else I could check to find out why?

**Reply by Lachney** [reply]
Whoop, I had to also modify the one in `sudo crontab -e` as well. Okay, I'm now at a new issue, the schedule is now stuck at "processing" and doesn't execute. Any ideas?

Running the ExecStart command manually, I am seeing this after the schedule is attempting to execute.
![image](https://user-images.githubusercontent.com/7856665/101992203-17065d80-3c77-11eb-8755-837b20301a11.png)

**Reply by Lachney** [reply]
After simply restarting the Minecraft server after making all those changes to the services, the schedules finally worked. Thanks.

**Reply by DaneEveritt** [reply]
Awesome, because I had no idea why it wasn't working. 😆
